test/onnx/exporter/test_core.py

```diff
@@ -15,17 +16,17 @@ class TorchTensorTest(common_utils.TestCase):
     @common_utils.parametrize(
         "dtype, np_dtype",
         [
-            (torch.bfloat16, np.uint16),
```

Comment:
I don't undeerstand why this is wrong and still passed the test?

Reply:
Is the test still legit?

Reply:
The test is changed with the implementation. So they are updated together to reflect the new behavior